docs/tools/hedgehog-engine/blueblur/files/ar.md

```diff
@@ -6,4 +6,9 @@ description: AR/ARL tools for Sonic Generations
 ## CompressAR
 Tool to compress and decompress AR files for Sonic Generations, using 'makecab' and 'expand'.
 
-[Download :fontawesome-solid-download:](https://github.com/PTKickass/CompressAR/releases){ .md-button .md-button--primary target="_blank"}
\ No newline at end of file
+[Download :fontawesome-solid-download:](https://github.com/PTKickass/CompressAR/releases){ .md-button .md-button--primary target="_blank"}
+
+## Generations Archive Editor
+Editor for Sonic Generations .ar.00/.pfd archives.
+
+[GitHub Repository :fontawesome-brands-github:](https://github.com/sonicretro/Generations-Archive-Editor){ .md-button target="_blank"}
```

Comment:
No need for this, you can remove it. Hedgehog Engine games already have a common archive editor.